examples/js/controls/MapControls.js

```diff
@@ -776,17 +775,29 @@ THREE.MapControls = function ( object, domElement ) {
 
 		switch ( event.button ) {
 
-			case scope.mouseButtons.ORBIT:
+			case scope.mouseButtons.LEFT:
 
-				if ( scope.enableRotate === false ) return;
+				if ( event.ctrlKey || event.metaKey ) {
```

Comment:
You should handle the control key separately. Right now when you press the ctrl key while already dragging the mouse, it doesn't change its state to rotate. Similarly, it does not stop rotating when you release the ctrl key while you keep dragging the mouse around.

Reply:
>when you press the ctrl key while already dragging the mouse, it doesn't change its state to rotate.

Why would we _prefer_ that behavior?

As currently implemented, the ctrl key must be pressed first.

Reply:
>> when you press the ctrl key while already dragging the mouse, it doesn't change its state to rotate.

> Why would we prefer that behavior?

That's how Google Maps behaves... 🤔

Reply:
Doesn't need to be addressed in this PR though...

Reply:
This can be merged, then. It maintains a similar API between `MapControls` and `OrbitControls`.

Reply:
Yeah, I think we should keep Orbit & Map core function as close as possible